bootstrap.example.toml

```diff
@@ -44,7 +44,6 @@
 # =============================================================================
 # Tweaking how LLVM is compiled
 # =============================================================================
-[llvm]
```

Comment:
Could you please add some note to the top of this file to:
1) Remind people to use the `section.key` format when adding things here.
2) Tell users that `llvm.foo = ` is equivalent `[llvm]\nfoo = ` (not everyone might have this TOML knowledge).